src/Lykke.Service.AssetDisclaimers.AzureRepositories/DisclaimerRepository.cs

```diff
@@ -14,6 +16,8 @@ public class DisclaimerRepository : IDisclaimerRepository
         private readonly INoSQLTableStorage<DisclaimerEntity> _storage;
         private readonly INoSQLTableStorage<AzureIndex> _disclaimerIdIndexStorage;
 
+        private readonly ConcurrentDictionary<string, List<Disclaimer>> _disclaimerListsByLykkeEntityId = new ConcurrentDictionary<string, List<Disclaimer>>();
```

Comment:
This makes service stateful.

Reply:
yes, now this service live in 1 copy. And for hotfix it should be enough.
If after release we will see that the situation has improved, then i make task for update this part and use addition table with index by disclaimer (or use redis), for avoid current situations. And this task will be performed by member of the support team in working mode.

Reply:
With the single instance PR is ok. Will approve when task is tested